### PR TITLE
fix: Use alert dialog to replace default dialog

### PR DESCRIPTION
**Pull request by yanguoyu**

### CODE REVIEW

**Review by FrederLu — APPROVED** [review]

**Comment by yanguoyu** [comment]
@devchenyan  @WhiteMinds @Keith-CY Please help me have a review.

File: packages/neuron-ui/src/containers/Main/hooks.ts [hunk 1]
```diff
@@ -294,13 +296,17 @@ export const useSubscription = ({
         }
       }
     })
+    const showGlobalDialogSubject = ShowGlobalDialogSubject.subscribe(params => {
```
Comment (Keith-CY) [inline packages/neuron-ui/src/containers/Main/hooks.ts:299]: I'l like to know why `show global dialog` and `show alert dialog` are paired

**Review by Keith-CY — APPROVED** [review]

File: packages/neuron-ui/src/containers/Main/hooks.ts (hunk 1 above)
Comment (yanguoyu) [reply packages/neuron-ui/src/containers/Main/hooks.ts:299]: Maybe renaming `showAlertDialog` to `showGlobalAlertDialog` is clear?

Comment (Keith-CY) [reply packages/neuron-ui/src/containers/Main/hooks.ts:299]: It would be more clear and won't mislead other developers in the future

**Review by WhiteMinds — APPROVED** [review]

**Pull request merged**